ci-operator/step-registry/openshift/image-based/upgrade/metal/config/openshift-image-based-upgrade-metal-config-commands.sh

```diff
@@ -31,6 +31,22 @@ cat <<EOF > ${SHARED_DIR}/install.sh
 #!/bin/bash
 set -euo pipefail
 
+function dnf_install_retry {
+  apps=(\$@)
+  echo "Apps I want to install are: \${apps[*]}"
+
+  for _ in \$(seq 3) ; do
+    sudo dnf clean -y all # clean the cache
+
+    # shellcheck disable=SC2086
+    sudo dnf install -y \${apps[*]} && return 0
```

Comment:
Will this behave correctly? I think because of that `-e` flag this will just fail and not loop, no?

Reply:
I tested by forcing trying to install a nonexistant package and it ran successfully (3 failed attempts)